components/molecules/LatestPrTableRow/latest-pr-table-row.tsx

```diff
@@ -37,47 +37,47 @@ const LatestPrTableRow = ({
   prUpdatedTime
 }: LatestPrTableRowProps) => {
   return (
-    <div className="flex gap-2 items-center px-2 py-1">
-      <div className={`flex cursor-default item-center gap-2 w-3/5 ${isHoverCard && "w-5/6"}`}>
-        {prStatus === "open" ? (
-          <IconContext.Provider value={{ color: "green", style: { width: 14, height: 14, marginTop: 2 } }}>
-            <VscGitPullRequest title="Open Pull Request" />
-          </IconContext.Provider>
-        ) : prStatus === "closed" ? (
-          <IconContext.Provider
-            value={{ color: merged ? "purple" : "red", style: { width: 14, height: 14, marginTop: 2 } }}
-          >
-            {merged ? (
-              <VscGitMerge title="Merged Pull Request" />
-            ) : (
-              <VscGitPullRequestClosed title="Closed Pull Request" />
-            )}
-          </IconContext.Provider>
-        ) : (
-          <IconContext.Provider value={{ color: "gray", style: { width: 14, height: 14, marginTop: 2 } }}>
-            <VscGitPullRequestDraft title="Draft Pull Request" />
-          </IconContext.Provider>
-        )}
-        <Text title="updated date">{calcDistanceFromToday(new Date(parseInt(prUpdatedTime, 10)))}</Text>
-        <Text title={prName} className="!text-light-slate-12 !w-32 md:!w-72  !truncate ">
-          <a href={`https://github.com/${repoOwner}/${repoName}/pull/${prNumber}`} target="_blank" rel="noreferrer">
+    <a href={`https://github.com/${repoOwner}/${repoName}/pull/${prNumber}`} className="!cursor-pointer" target="_blank" rel="noreferrer">
```

Comment:
This is not correct, we can't wrap all this content in an inline anchor, at best the link would be programatic; even so, the content this encapsulates requires does not appear to be inline.

What was the reasoning behind making wrapping everything in an `<a>`?

Reply:
The entire pr list item should be clickable and linked to the PR. 
What do you suggest instead?

Reply:
Maybe we end up walking this back. Making the title clickable only is fine. The title is not clickable in production or is inconsistent. Let's not wrap the whole div and focus on the title work.

https://www.loom.com/share/c8fe0a4467b94548ba50af992b42421d

New goal
1. Make the title clickable
2. Add cursor on the link